packages/lib/modules/pool/actions/LiquidityActionHelpers.ts

```diff
@@ -412,12 +427,14 @@ export function formatBuildCallParams<T>(buildCallParams: T, account: Address) {
 }
 
 export function toTokenAmountsIn(
-  sdkQueryOutput: AddLiquidityQueryOutput
+  sdkQueryOutput: AddLiquidityQueryOutput,
+  pool: Pool
 ): TokenAmountIn[] | undefined {
   if (!sdkQueryOutput) return
   return sdkQueryOutput.amountsIn.map(amountIn => ({
     address: amountIn.token.address,
     amount: amountIn.amount,
+    symbol: amountIn.token.symbol || getActionableTokenSymbol(amountIn.token.address, pool),
```

Comment:
Why not just a more generic function which searches all the pool's token for a
matching token to get the symbol? e.g. `getPoolTokenSymbol(address, pool)`

Reply:
I prefer to wait until we do the next refactor that clearly splits between 3 sets of tokens (Header/composition/liquidity actions). (added a TODO and ticket for that):
https://github.com/balancer/frontend-monorepo/blob/a898f5ab1b14c951b11cf394cf9476306249d6e3/packages/lib/modules/pool/pool.helpers.ts#L424